flink-python/pyflink/fn_execution/coders.py

```diff
@@ -38,6 +39,38 @@
            'TimestampCoder', 'ArrayCoder', 'MapCoder', 'DecimalCoder']
 
 
+class TableFunctionRowCoder(FastCoder):
+    """
+    Coder for Table Function Row.
+    """
+    def __init__(self, row_coder):
```

Comment:
How about rename row_coder to flatten_row_coder? It is more clear to distinguish flatten_row_coder from row_coder